nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/storage/utils/AzureStorageUtils.java

```diff
@@ -311,4 +316,34 @@ public static void setProxy(final OperationContext operationContext, final Proce
         final ProxyConfiguration proxyConfig = ProxyConfiguration.getConfiguration(processContext);
         operationContext.setProxy(proxyConfig.createProxy());
     }
+
+    public static void configureProxy(final NettyAsyncHttpClientBuilder nettyClientBuilder, final PropertyContext propertyContext) {
```

Comment:
That's a good point @turcsanyip, I agree that it would be better to decouple this utility method from the Netty Client implementation.  This PR does couple the components to the Netty implementation, where previously the implementation was loaded based on available libraries. Not sure of other impacts, but something to consider.

Reply:
Hey! I just wanted to follow a pattern that I've seen in the class. (`AbstractAzureBlobProcessor_v12.configureCredential()`) But apart from that I agree with you. Changed it!